pkg/components/versions.go

```diff
@@ -16,51 +16,51 @@ package components
 
 // This section contains images used when installing open-source Calico.
 const (
-	VersionCalicoNode            = "v3.11.1-with-auto-backend"
-	VersionCalicoCNI             = "v3.11.1"
-	VersionCalicoTypha           = "v3.11.1"
-	VersionCalicoKubeControllers = "v3.11.1"
-	VersionFlexVolume            = "v3.11.1"
+	VersionCalicoNode            = "v3.12.0"
+	VersionCalicoCNI             = "v3.12.0"
+	VersionCalicoTypha           = "v3.12.0"
+	VersionCalicoKubeControllers = "v3.12.0"
+	VersionFlexVolume            = "v3.12.0"
 )
 
 // This section contains images used when installing Tigera Secure.
 const (
 	// Overrides for Calico.
-	VersionTigeraNode            = "v2.7.0-0.dev-188-gc5255d3"
-	VersionTigeraTypha           = "v2.7.0-0.dev-74-g67bc0b1"
-	VersionTigeraKubeControllers = "v2.7.0-0.dev-109-gfa8b79c"
+	VersionTigeraNode            = "v2.7.0"
+	VersionTigeraTypha           = "v2.7.0"
+	VersionTigeraKubeControllers = "v2.7.0"
 
 	// API server images.
-	VersionAPIServer   = "v2.7.0-0.dev-32-gb094281f"
```

Comment:
Is this right? I think the existing v2.7.0-0.dev tags are more recent than the v2.7.0 release.

Reply:
Looking at the hash, `b094281f` is older than the tagged `v2.7.0`. So I think it is correct to update those.

Reply:
Oh, ok. yeah I think I misread the tag. :+1: